iterative/aws/provider.go

```diff
@@ -169,6 +180,10 @@ func ResourceMachineCreate(ctx context.Context, d *schema.ResourceData, m interf
 				Name:   aws.String("vpc-id"),
 				Values: []*string{aws.String(vpcID)},
 			},
+			{
+				Name:   aws.String("available-ip-address-count"),
+				Values: []*string{aws.String(vpcID)},
+			},
```

Comment:
What does this do? After reading [the documentation](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeSubnets.html), it appears like you're searching for subnets that have a given number of addresses, but that number seems to be the identifier of the VPC. 🤔  Could it be a copy-paste mistake?

Reply:
This is not needed and indeed no test done by me was done with this section, if this work we wont have any VPC unless  the id coincides with the number of ips!

Reply:
I think that VScode added that while saving from a previous atempt? No clue

Reply:
```suggestion
```

I barely can't refrain myself from creating a new [humorous unit of measurement](https://en.wikipedia.org/wiki/List_of_humorous_units_of_measurement), the MiracleBlock.